es-ES/mail/chrome/messenger/messenger.properties

```diff
@@ -424,10 +424,12 @@ editVirtualFolderPropertiesTitle=Editar las propiedades de la búsqueda %S
 #alert to inform the user to choose one or more folders to search for a saved search folder
 alertNoSearchFoldersSelected=Debe elegir al menos una carpeta en la que buscar para guardar la búsqueda.
 
-# These two are displayed in the message and folder pane windows
-# LOCALIZATION NOTE(????ByteAbbreviation): Do not translate %d below, it is the size of the message/folder
-kiloByteAbbreviation=%dKB
-megaByteAbbreviation=%dMB
+# These are displayed in the message and folder pane windows
+# LOCALIZATION NOTE %.*f is the abbreviated size in the appropriate units
+byteAbbreviation2=%.*f bytes
```

Comment:
Note that all messenger.properties files seem to do something different. There was an issue with es-ES using the old method.  The attached is what master looks like:

<img width="342" alt="screen shot 2017-03-23 at 8 54 51 am" src="https://cloud.githubusercontent.com/assets/7443120/24256838/70c38eee-0fa6-11e7-8faf-d8055d8ab95d.png">